shrine/lib/colors.dart

```diff
@@ -14,15 +14,15 @@
 
 import 'package:flutter/material.dart';
 
-const kShrinePink50 = const Color(0xFFFEEAE6);
-const kShrinePink100 = const Color(0xFFFEDBD0);
-const kShrinePink300 = const Color(0xFFFBB8AC);
-const kShrinePink400 = const Color(0xFFEAA4A4);
+const kShrinePink50 = Color(0xFFFEEAE6);
```

Comment:
Will Dart still infer these to be constants?

Reply:
The second `const` is not needed in Dart 2 IIUC, and this change is encouraged by the `unnecessary_const` lint check.